packages/shared/src/components/Feed.spec.tsx

```diff
@@ -698,23 +691,19 @@ it('should report broken link with comment', async () => {
       },
     },
   ]);
-  const [menuBtn] = await screen.findAllByLabelText('Options');
-  menuBtn.click();
-  const contextBtn = await screen.findByText('Report');
-  contextBtn.click();
-  const brokenLinkBtn = await screen.findByText('Broken link');
-  brokenLinkBtn.click();
-  const input = (await screen.findByRole('textbox')) as HTMLTextAreaElement;
-  fireEvent.change(input, { target: { value: 'comment' } });
-  input.dispatchEvent(new Event('input', { bubbles: true }));
-  const submitBtn = await screen.findByText('Submit report');
-  submitBtn.click();
-
-  await waitFor(async () => {
-    await screen.findByRole('alert');
-    const feed = await screen.findByTestId('posts-feed');
 
-    return expect(feed).toHaveAttribute('aria-live', 'assertive');
+  await act(async () => {
+    const [menuBtn] = await screen.findAllByLabelText('Options');
+    menuBtn.click();
```

Comment:
Would not the correct way be to utilize `fireEvent` which wraps with `act` automatically, we did this to fix some flaky tests before.

Reply:
Ah yeah good point, can adjust.
Either way think these will work, but yes maybe cleaner